fuzzing/src/utils/noncopyable.h

```diff
@@ -0,0 +1,31 @@
+// noncopyable.h
+//
+//   Helper base class for copy constructor and copy assignment
+//   operator deletion.
+//
+// Copyright 2020 by
+// Dominik Roettsches
+//
+// This file is part of the FreeType project, and may only be used,
+// modified, and distributed under the terms of the FreeType project
+// license, LICENSE.TXT.  By continuing to use, modify, or distribute
+// this file you indicate that you have read the license and
+// understand and accept it fully.
+
+#ifndef UTILS_NONCOPYABLE_H_
+#define UTILS_NONCOPYABLE_H_
+
+namespace freetype
+{
+  class noncopyable
+  {
+    protected:
+      noncopyable()  = default;
+      ~noncopyable() = default;
+
+      noncopyable( const noncopyable& ) = delete;
+      noncopyable& operator=( const noncopyable& ) = delete;
+  };
+}  // namespace freetype
```

Comment:
thanks a lot for this! 😊 qq -- can you please match the code style with e.g. https://github.com/freetype/freetype2-testing/blob/master/fuzzing/src/utils/tarreader.h;

- empty lines (sometimes FT uses two empty lines consecutively)
- indentation (`protected` should sit on the same level as `class`)
- the return type should sit on it's own line
- happy to keep the end-comment on the `namespace` 😊

Reply:
Done